react-ai-chatbot/src/components/Chat/Chat.module.css

```
@@ -16,6 +16,7 @@
     &[data-role="user"] {
         align-self: flex-end;
         background: #f3f3f3;
+        background: light-dark(#f3f3f3, #303030);
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

background is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)